src/pattern-library/routes.ts

```diff
@@ -196,10 +196,10 @@ const routes: PlaygroundRoute[] = [
     route: '/input-option-button',
   },
   {
-    title: 'SelectNext',
+    title: 'Selects',
```

Comment:
```suggestion
    title: 'Select',
```

All the other menu items are named after the main component on that page, so I think we should do the same here.

Reply:
I used the plural form because that's what we do for `Dialog` and `ModalDialog` elements. The page's title is `Dialogs` and then there are two sub-sections for them.